code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -217,12 +217,12 @@ can cause issues with ammo types getting mixed up during the burst.
 
 
 /obj/item/weapon/gun/shotgun/combat/set_gun_config_values()
-	fire_delay = config.mhigh_fire_delay*2
-	accuracy_mult = config.base_hit_accuracy_mult
-	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.hmed_hit_accuracy_mult
+	fire_delay = config.high_fire_delay * 3 //one shot every 1.5 seconds.
+	accuracy_mult = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult
+	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.max_hit_accuracy_mult //you need to wield this gun for any kind of accuracy
```

Comment:
Simplify this by doing `config.base_hit_accuracy_mult - config.max_hit_accuracy_mult` instead.

Reply:
This is the convention for most weapons with an accuracy bonus of some kind.

Reply:
It makes for a convoluted system. To be fair this could use a general rework.

Reply:
Because it is resolved

Reply:
The issue persists. Get a different maintainer if you disagree with my request.